go.mod

```diff
@@ -1,37 +1,39 @@
 module github.com/grafana/xk6-websockets
 
-go 1.20
+go 1.21
+
+toolchain go1.22.3
```

Comment:
Can we revert this as it will propagate through everything? 

I am all for us to go do it in k6 ... soon - probably next cycle, but doing it first in the extensions seems like a bad idea.